src/service.rs

```diff
@@ -1062,12 +1097,14 @@ pub(crate) async fn handle_received_txns<Types: NodeType>(
     let time_in = Instant::now();
     for tx in txns.into_iter() {
         let commit = tx.commit();
+        let len = bincode::serialized_size(&tx).unwrap_or_default();
```

Comment:
This is the only method of estimating transaction's size short of actually serializing it we have. We should add one in HotShot.